Code/Engine/Foundation/Platform/Android/Screen_Android.cpp

```diff
@@ -1,11 +1,22 @@
 #include <Foundation/FoundationPCH.h>
 
-#if EZ_ENABLED(EZ_PLATFORM_ANDROID)
+#include <Foundation/Basics/Platform/PlatformFeatures.h>
+#include <Foundation/System/Screen.h>
 
-#  if EZ_ENABLED(EZ_SUPPORTS_GLFW)
-#    include <Foundation/Platform/GLFW/Screen_GLFW.h>
-#  else
-#    include <Foundation/Platform/NoImpl/Screen_NoImpl.h>
-#  endif
+#if EZ_ENABLED(EZ_PLATFORM_ANDROID)
 
+ezResult ezScreen::EnumerateScreens(ezHybridArray<ezScreenInfo, 2>& out_Screens)
```

Comment:
We could change this function to just take a dynamicarray.

Reply:
Will do!